pkg/iptables/iptables.go

```diff
@@ -25,6 +25,7 @@ const (
 
 var (
 	supportsXlock       = false
+	iptablesPath        = ""
```

Comment:
I am not a big fan of this but I see what you are trying to do. It just might be confusing to someone just looking at the code with no background